### ISSUE TITLE
Build errors when building from source

### ISSUE THREAD

**Issue by alexander-poulikakos** [VRL-Studio/VRL-Studio#43]
I want to build VRL-STUDIO from source. Following the `How To Build` steps in the [README](https://github.com/VRL-Studio/VRL-Studio). 

I have downloaded Apache NetBeans IDE version 12.2 (First time I'm using this IDE so I might be missing something simple...).

It fails when building VRL library. Seems to not find some dependencies? How to fix?
```
...
Compiling 1081 source files to /Users/alexanderpoulikakos/git/VRL/VRL/build/classes
/Users/alexanderpoulikakos/git/VRL/VRL/src/eu/mihosoft/vrl/v3d/VGeometry3D.java:63: error: package javafx.scene.paint does not exist
import javafx.scene.paint.PhongMaterial;
/Users/alexanderpoulikakos/git/VRL/VRL/src/eu/mihosoft/vrl/v3d/VTriangleArray.java:62: error: package javafx.scene.shape does not exist
import javafx.scene.shape.TriangleMesh;
...
```

See full log/Output here: https://gist.github.com/alexander-poulikakos/550fc7f44abe189c5a91b913c0916720

![image](https://user-images.githubusercontent.com/16044466/111341329-b4f43480-8679-11eb-9b37-6b4b8cd1205a.png)

**Comment by stephanmg** [comment]
Hi @alexander-poulikakos,

it looks like you haven't installed JavaFX. Do you use an older version of the JDK? If so, you might need to install a newer version.

**Comment by alexander-poulikakos** [comment]
Yes, I have Java 1.8.
![image](https://user-images.githubusercontent.com/16044466/111342410-bbcf7700-867a-11eb-8f5d-dd86b4354192.png)

Maybe I need to enable it somehow?

**Comment by stephanmg** [comment]
JavaFX libraries are usually not shipped standard with most OpenJDK or Oracle JDK versions.

**Comment by alexander-poulikakos** [comment]
I use `jdk1.8.0_271.jdk` as downloaded (I while ago) from  https://www.oracle.com/java/technologies/javase/javase-jdk8-downloads.html

**Comment by alexander-poulikakos** [comment]
Seems I have some sort of broken reference. I'll try to figure out what it means.
![image](https://user-images.githubusercontent.com/16044466/111343349-a018a080-867b-11eb-86c9-55959d3c4e84.png)

**Comment by stephanmg** [comment]
Yeah I think that Oracle JDK does not ship with JavaFX. What OS are you using? For me it was as easy as `apt-get install openjfx` on Ubuntu.

**Comment by alexander-poulikakos** [comment]
Mac OSX

**Comment by alexander-poulikakos** [comment]
I have switched to an older version of Netbeans: 8.2
and now it works.
